server/impl/src/main/java/com/walmartlabs/concord/server/process/pipelines/processors/ConfigurationProcessor.java

```diff
@@ -223,4 +229,14 @@ private static void processHandlersConfiguration(Payload payload, Map<String, Ob
             m.put(Constants.Request.PROCESS_TIMEOUT, handlerTimeout);
         }
     }
+
+    private void processDryRunModeConfiguration(Payload payload, Map<String, Object> m) {
+        Object dryRunMode = m.get(Constants.Request.DRY_RUN_MODE_KEY);
+        if (dryRunMode instanceof String) {
+            dryRunMode = Boolean.parseBoolean((String) dryRunMode);
+            m.put(Constants.Request.DRY_RUN_MODE_KEY, dryRunMode);
+        }
+
+        logManager.info(payload.getProcessKey(), "Dry run mode: {}", MapUtils.getBoolean(m, Constants.Request.DRY_RUN_MODE_KEY, false));
```

Comment:
On the one hand, there will be fewer logs; on the other hand, it's unclear whether it wasn't logged because it never gets logged at all or if something is wrong with the parameter...